### DISCUSSION TITLE
How to squeeze omr?

### DISCUSSION THREAD

**Discussion by rdmitry0911** [Ysurac/openmptcprouter#3178] [Q&A]
Hi guys, I'm trying to build omr for [MOFI5500-5GXeLTE em9191](https://mofinetwork.com/files/MoFi_Network_MOFI5500_5GXeLTE_EM9191_SPECS.pdf) router. It seems to be capable to run omr. It has 512MB RAM and MediaTek MT7621 4 cores cpu. However it has only 28MB of flash and omr doesn't fit into it. I tried to create an image selecting omr-minimal, but its size is about 43MB and is still too big. A working openwrt image for this device is only about 8MB. I think there is a way to remove unneeded parts from the image but I don't know to make it without a risk to remove also some of the required parts. 

Any help would be very much appreciated

**Comment by Ysurac** [comment]
You can select packages you don't want in menuconfig, remove openmtpcprouter-full as it's force others packages (and openmptcprouter-mini is not updated), and remove xray and v2ray packages because they are huge.
MT7621 is a really slow CPU, don't expect too much from that...
